docs/feature/1665-custom-sharding.md

```diff
@@ -220,6 +220,109 @@ You can now start a flank test run. With the updated config there will still be
     * `debug-2.apk` with 2 shards
```

Comment:
Can you update header of this file:?
##### NOTE: Currently for android only, iOS support will be released soon